draft-ietf-httpbis-http2bis.xml

```diff
@@ -2667,7 +2663,7 @@ CONTINUATION Frame {
         </t>
         <t>
           CONTINUATION frames MUST be associated with a stream. If a CONTINUATION frame is received
-          with a Stream Identifier field of 0x00, the recipient MUST respond with a <xref target="ConnectionErrorHandler">connection error</xref> of type PROTOCOL_ERROR.
+          whose Stream Identifier field is 0x00, the recipient MUST respond with a <xref target="ConnectionErrorHandler">connection error</xref> of type PROTOCOL_ERROR.
```

Comment:
I think we should revert this too.  "whose" implies unnecessary anthropomorphism, which is worth avoiding.

```suggestion
          with a Stream Identifier field of 0x00, the recipient MUST respond with a <xref target="ConnectionErrorHandler">connection error</xref> of type PROTOCOL_ERROR.
```